conformance/system_macros/make_timestamp.ion

```diff
@@ -1,22 +1,305 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-// Test Cases:
-// make_timestamp can be invoked using any type of macro reference.
-// make_timestamp creates a single unannotated timestamp
-// any annotations on the argment values are silently dropped
-// if any argument is null, it should signal an error
-// year must be from 0 to 9999 (inclusive)
-// month must be from 1 to 12 (inclusive)
-// if month is present, year must also be present
-// day must be from 1 to 28,29,30, or 31 (inclusive) depending on the month
-// if day is present, month must also be present
-// hour must be from 0 to 23 (inclusive)
-// if hour is present, minute and day must also be present
-// minute must be from 0 to 59 (inclusive)
-// if minute is present, hour must also be present
-// second must be a decimal value from +0 to 60 (exclusive) with any precision
-// if second is present, minute must also be present
-// offset minutes argument may not be present unless minutes is present.
-// if offset minutes is present, it must be in the range -1440 to 1440 (inclusive)
-// if offset minutes is absent, the resulting timestamp has unknown offset.
+(ion_1_1 "make_timestamp can be invoked"
+         (each "in text with an unqualified macro name"
+               (text " (:make_timestamp 1) ")
+               "in text with an unqualified macro address"
+               (text " (:7 1) ")
+               "in text with a qualified macro name"
+               (text " (:$ion::make_timestamp 1) ")
+               "in text using qualified system macro address 7"
+               (text " (:$ion::7 1) ")
+               "in binary using system macro address 7"
+               (binary "EF 07 00 00 01 00")
```

Comment:
Interesting, `make_timestamp` is the only system macro that requires a 2-byte arg encoding bitmap. I think this is probably ok since most of its value proposition comes from usage in TDL, but it is eye-catching.